WooCommerce/Classes/System/SessionManager.swift

```diff
@@ -288,7 +295,7 @@ private extension SessionManager {
 
     /// Updates the timestamps that control when background data is fetched.
     ///
-    @objc func handleStorageDidReset() {
+    @objc func handleStorageDidInvalidateData() {
```

Comment:
nit: how about marking `resetTimestampsValues` as `@objc`, removing this function, and calling `resetTimestampsValues` from the two notifications instead? I wasn't very sure the "invalidate data" part in the function name.

Reply:
Good idea! Done in 56c7182f69a2b6f488bd557ab154dbe9e3a990ab.